tsbridge/metric.go

```diff
@@ -38,6 +40,20 @@ type Metric struct {
 	Record    storage.MetricRecord
 }
 
+// UpdateMetrics contains all external metric dependencies
+type UpdateMetrics struct {
+	SDClient       StackdriverAdapter
```

Comment:
[Apologies, I realize I suggested UpdateMetrics in a previous comment, but reading through it again, I've added another commend. Sorry for making you revise these again].

readability nit:

The name of the struct should reflect what it holds. `UpdateMetrics` is probably a name better suited for a func that does the updates.

Prefer something like:

```
type Metrics struct {
 SDClient       StackdriverAdapter
 StatsCollector *StatsCollector
}
```

Then your func that does the actual update could be along the lines of `func (m *Metrics) UpdateAll(.....`

Reply:
I went with Metrics struct name here.